contracts/liquidator/Liquidator.sol

```diff
@@ -126,51 +154,70 @@ contract Liquidator is IUniswapV3FlashCallback, PeripheryImmutableState, Periphe
     function setPoolConfigs(
         address[] memory _assets,
         bool[] memory _lowLiquidityPools,
-        uint24[] memory _poolFees
+        uint24[] memory _poolFees,
+        Exchange[] memory _exchanges
     ) external {
         if (msg.sender != admin) revert Unauthorized();
 
         for (uint8 i = 0; i < _assets.length; i++) {
             address asset = _assets[i];
             bool lowLiquidity = _lowLiquidityPools[i];
             uint24 poolFee = _poolFees[i];
-            poolConfigs[asset] = UniswapPoolConfig({isLowLiquidity: lowLiquidity, fee: poolFee});
+            Exchange exchange = _exchanges[i];
+            poolConfigs[asset] = PoolConfig({
+                isLowLiquidity: lowLiquidity,
+                fee: poolFee,
+                exchange: exchange
+            });
         }
     }
 
     /**
      * @dev Returns Uniswap pool config for given asset
      */
-    function getPoolConfigForAsset(address asset) internal view returns(UniswapPoolConfig memory) {
-        UniswapPoolConfig memory config = poolConfigs[asset];
+    function getPoolConfigForAsset(address asset) internal view returns(PoolConfig memory) {
+        PoolConfig memory config = poolConfigs[asset];
         // If asset is not found, proceed with default pool config
         if (config.fee == 0) {
-            return UniswapPoolConfig({
+            return PoolConfig({
                 fee: DEFAULT_POOL_FEE,
-                isLowLiquidity: false
+                isLowLiquidity: false,
+                exchange: Exchange.Uniswap
             });
```

Comment:
IMO, we should probably just revert instead of swapping through some pool that will likely either revert or eat up our profits.

Reply:
But we don't have to make that change in this PR.

Reply:
Actually, your `amountOutMin` change probably takes care of this issue anyways